build.gradle.kts

```diff
@@ -253,7 +253,8 @@ githubRelease {
     prerelease.set(isDevelopmentRelease)
     overwrite.set(isDevelopmentRelease)
     generateReleaseNotes.set(false)
-    body.set(layout.projectDirectory.file("release/changes.md").asFile.readText().trim())
+    body.set(releaseNotes())
```

Comment:
IIRC in the other PRs you added a `releaseNotes` variable at the top of the script.. I would do the same so that everything is consistent.

Reply:
Can do. The reason I did not do that here is that it is only used in one place (to set the `githubRelease` body).